news/2025/2025-09-12-the-state-of-ranked-results.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: "The State of Ranked: Results"
```

Comment:
to make it clear that this is not a tournament or event or smth
with some some people thinking that cta was a regular tournament, i'd expect that some people have forgotten about (or never heard about) this "state of ranked" think altogether

```suggestion
title: "The State of Ranked: Survey Results"
```

Reply:
Eh, will apply just in case, but the banner should be a dead giveaway. Plus people would quickly realize this is anything but an event on a quick first look.